database/seeders/BoardSeeder.php

```diff
@@ -134,12 +161,19 @@ public function run()
                     ]);
                 
                 //People
+                    //icons
+                        $attractive_url = Storage::disk('public')->url('Icons/attractive.png');
+                        //kind
+                        //mean
+                        //beautiful
+                        //handsome
+                        //goregous
                     $user->words()->createMany([
                         ['id' => $adjectives+12, 'text' => 'Kind', 'color' => $default_color],
                         ['id' => $adjectives+13, 'text' => 'Mean', 'color' => $default_color],
-                        ['id' => $adjectives+14, 'text' => 'Gay', 'color' => $default_color],
-                        ['id' => $adjectives+15, 'text' => 'Queer', 'color' => $default_color],
-                        ['id' => $adjectives+16, 'text' => 'Attractive', 'color' => $default_color],
+                        ['id' => $adjectives+14, 'text' => 'Gay', 'color' => $default_color, 'icon' => $rainbow_url],
+                        ['id' => $adjectives+15, 'text' => 'Queer', 'color' => $default_color, 'icon' => $rainbow_url],
+                        ['id' => $adjectives+16, 'text' => 'Attractive', 'color' => $default_color, 'icon' => $attractive_url],
                         ['id' => $adjectives+17, 'text' => 'Beautiful', 'color' => $default_color],
                         ['id' => $adjectives+18, 'text' => 'Handsome', 'color' => $default_color],
                         ['id' => $adjectives+19, 'text' => 'Goregous', 'color' => $default_color]
```

Comment:
```suggestion
                        ['id' => $adjectives+19, 'text' => 'Gorgeous', 'color' => $default_color]
```
I think there is a typo here for "Gorgeous".